programs/issue-2-temp-calculator/run.py

```diff
@@ -0,0 +1,39 @@
+def celsius_to_fahrenheit(celsius):
+    fahrenheit = celsius * 9/5 + 32
+    print("{}°C to {}°F".format(celsius, fahrenheit))
+
+
+def fahrenheit_to_celsius(fahrenheit):
+    celsius = (fahrenheit - 32) * 5/9
+    print("{}°F to {}°C".format(fahrenheit, celsius))
+
+
+def menu():
+    print("Program do konwersji temperatury w różnych skalach. Jaką konwersje chcesz wykonać?")
+    print("1) Stopnie Celsjusza na Fahrenheita")
+    print("2) Stopnie Fahrenheita na Celsjusza")
+    while True:
+        try:
+            user_request = int(input("Wybierz: "))
+        except ValueError:
+            print("Musisz podać numer opcji!")
+            continue
+        if not 2 >= user_request >= 1:
+            print("Nie istnieje taka opcja.")
+            continue
+        else:
+            break
+    while True:
+        try:
+            value = float(input("Podaj wartość: "))
+        except ValueError:
+            print("Musisz podać liczbę!")
+            continue
+        break
+    if user_request == 1:
+        celsius_to_fahrenheit(value)
+    elif user_request == 2:
+        fahrenheit_to_celsius(value)
+
+
+menu()
```

Comment:
dobrym zwyczajem jest zostawiać na końu zawsze pustą linię. 

![Selection_999(212)](https://user-images.githubusercontent.com/201263/64439277-e3c73d00-d0c9-11e9-9318-5a5cf3dee041.png)

Ustaw sobie swój edytor, by sam tego pilnował i dodawał ewentualnie brakującą linie, gdyby jej brakowało.

A dlaczego warto  jest dodawać nową linię? Bo jeżeli kiedyś coś nowego dodasz do następnej lini, to wtedy w diffie będzie widoczna zmiana tej nowej lini. A tak to byś widział zmiany w dwóch liniach. Dopisanie znaku nowej lini do lini 39, tuż za `menu()` i nowa linia.

I tym samym git blame pokazałby wówczas, że to ty zmieniłeś obie linie.